src/main/resources/edu/cornell/kfs/sys/businessobject/datadictionary/DocumentHeader.xml

```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<beans xmlns="http://www.springframework.org/schema/beans"
+    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xmlns:p="http://www.springframework.org/schema/p"
+    xsi:schemaLocation="http://www.springframework.org/schema/beans http://www.springframework.org/schema/beans/spring-beans.xsd">
+
+    <bean id="DocumentHeader-documentDescription"
+        parent="DocumentHeader-documentDescription-parentBean">
+        <property name="validationPattern">
+            <bean parent="AnyCharacterValidationPattern" p:allowWhitespace="true" p:maxLength="40" />
```

Comment:
Can the `maxLength` setting be removed from the validation pattern? That setting should already be present at the AttributeDefinition level for these various beans.